tests/covariance_matrix_05.cc

```diff
@@ -0,0 +1,67 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2020 by the SampleFlow authors.
+//
+// This file is part of the SampleFlow library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+
+// Check the CovarianceMatrix consumer with complex numbers.
+// This test generates a simple samples that only have one component
+// (so we are really only calculating variance, not covariance) which
+// is a complex number with no real component.
```

Comment:
This is something that is at least not obvious to me. Could you add a line or two to the comment here that explains what samples you use, and why the variance for these should really be an imaginary number?

Reply:
I agree that it is confusing. I'm using the word "component" to mean two different things. I've update the comments at the top of each test. But the variance should in fact be a real number, not an imaginary number.